sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -448,9 +452,18 @@ export class EventHubConsumer {
         ? retryOptions.retryInterval / 1000
         : Constants.defaultDelayBetweenOperationRetriesInSeconds;
 
+    const timeoutInMs = retryOptions.timeoutInMs;
+    const retryPolicy = retryOptions.retryPolicy;
+    const minExponentialRetryDelayInMs = retryOptions.minExponentialRetryDelayInMs;
```

Comment:
Are there any default values for the new exponential retry fields you're adding? Will `retry` throw any errors if they are missing but we've specified an `exponential` retry policy?

Reply:
Yeah, default values are being set https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-amqp/src/retry.ts#L160-L171
So nope, no errors will be thrown if just the the `retryPolicy` is selected to be as exponential